src/yoroi-wallets/cardano/byron/constants.ts

```diff
@@ -0,0 +1,124 @@
+import {DefaultAsset, TokenInfo} from '../../types'
+
+export * from '../../cardano/constants'
+
+export const NETWORK_ID = 1
+export const WALLET_IMPLEMENTATION_ID = 'haskell-byron'
+export const PURPOSE = 2147483692 // BIP44
+export const COIN_TYPE = 2147485463
+export const ACCOUNT_INDEX = 0
+export const HARD_DERIVATION_START = 2147483648
+export const CHAIN_NETWORK_ID = '1'
+export const BIP44_DERIVATION_LEVELS = {
+  ROOT: 0,
+  PURPOSE: 1,
+  COIN_TYPE: 2,
+  ACCOUNT: 3,
+  CHAIN: 4,
+  ADDRESS: 5,
+} as const
+export const KEY_DEPOSIT = '2000000'
+export const POOL_DEPOSIT = '500000000'
+export const LINEAR_FEE = {
+  COEFFICIENT: '44',
+  CONSTANT: '155381',
+} as const
+export const MINIMUM_UTXO_VAL = '1000000'
```

Comment:
Also it's better to add `coins_per_utxo_size` parameter. The parameter is used by min_ada_for_output. But be careful because there is also old parameter called as `coins_per_utxo_word`.  `coins_per_utxo_word = coins_per_utxo_size * 8` and `coins_per_utxo_size = Math.Floor(coins_per_utxo_word / 8) `

Reply:
Current value you can get from https://cexplorer.io/params 
now `coins_per_utxo_size`  is 4310